src/Footer.tsx

```diff
@@ -30,6 +31,9 @@ export const Footer: React.SFC<{}> = () => {
         <div id="footer-logo-link">{logo}</div>
       </div>
       <div id="portal-contact-footer" className="center-content">
+        <span className={"bootstrap-4-backport experimental-mode"}>
+          <ExperimentalMode />
```

Comment:
Looking at this in the context of a portal footer makes me think of a few things.
1.  Should we text transform this so that it's all uppercase (like all other items in the portal footer)?
2. The meaning might be unclear for the user.  Could you show a tooltip on hover?  Can have the same popover info as synapse.org:

> This mode gives you early access to features that are still in development. Please note that we do not guarantee an absence of errors, and that the data created using these features may be lost during product upgrade.

Reply:
Upgrading SRC to release 1.16.0 will pick up the tooltip and uppercase changes in this [PR](https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/1037).